tests/smoketest/smoketest.sh

```diff
@@ -130,17 +127,24 @@ echo "*** [INFO] Logs from alertmanager..."
 oc logs "$(oc get pod -l app=alertmanager -o jsonpath='{.items[0].metadata.name}')" -c alertmanager
 echo
 
-echo "*** [INFO] Cleanup resources..."
-if $CLEANUP; then
-    oc delete "job/stf-smoketest-${NAME}"
-fi
+# moved this check to near the end as previously noted the delivery of this alert and the resulting trap delivery does take some time
+echo "*** [INFO] Check status of SNMP webhook trap delivery"
+SNMP_WEBHOOK_POD=$(oc get pod -l "app=default-snmp-webhook" -ojsonpath='{.items[0].metadata.name}')
+oc logs "$SNMP_WEBHOOK_POD" | grep 'Sending SNMP trap'
+SNMP_WEBHOOK_STATUS=$?
 echo
 
 if [ $SNMP_WEBHOOK_STATUS -ne 0 ]; then
     echo "*** [FAILURE] SNMP Webhook failed"
     exit 1
 fi
```

Comment:
This might not even still be enough. In my other testing (https://github.com/infrawatch/service-telemetry-operator/pull/379/files#diff-16335c82c44304ee49855ecebf854c2a8db384a1240f299c5f023b4b9845e407R135-R151) I placed the checks after the clean up, which itself probably has some extra delay built in. It might make more sense to loop this with set of timeouts and max loop count before deciding to ultimately fail this.